keras/preprocessing/image.py

```diff
@@ -227,6 +227,8 @@ class ImageDataGenerator(object):
         rescale: rescaling factor. If None or 0, no rescaling is applied,
             otherwise we multiply the data by the value provided
             (before applying any other transformation).
+        preprocess_function: function that will be implied on each input.
```

Comment:
fixed, but I think the input is one image (numpy tensor with rank 3)
looks like standardize function input (which preprocessing_function use as well) is one image (also written in a comment inside the function)